src/console/mirrorManager.ts

```diff
@@ -90,7 +90,11 @@ export class MirrorManager extends EventEmitter {
       connection.on(ConnectionEvent.HANDSHAKE, (details: ConnectionDetails) => {
         this.emit(MirrorEvent.LOG, "Got handshake from wii");
         this.emit(MirrorEvent.LOG, details);
-        fileWriter.updateSettings({ consoleNickname: details.consoleNick });
+
+        const replayFolder = path.join(config.folderPath, details.consoleNick);
+        fs.ensureDirSync(replayFolder);
```

Comment:
I'm not sure I like the idea of forcing the file to always be in the nickname folder. I think it should be an option but not mandated. Can we come up with an alternative option?